protostar/starknet_gateway/gateway_facade.py

```diff
@@ -19,6 +18,8 @@
 from protostar.starknet_gateway.starknet_request import StarknetRequest
 from protostar.utils.log_color_provider import LogColorProvider
 
+GatewayFacadeSupportedInputType = Union[List[int], Dict[str, Any]]
```

Comment:
I dont quite understand what this tyle describes from its name

Reply:
Renamed